src/main/java/com/powsybl/openloadflow/network/impl/LfHvdcImpl.java

```diff
@@ -10,6 +10,7 @@
 import com.powsybl.iidm.network.HvdcLine;
 import com.powsybl.iidm.network.extensions.HvdcAngleDroopActivePowerControl;
 import com.powsybl.iidm.network.extensions.HvdcOperatorActivePowerRange;
+import com.powsybl.openloadflow.ac.equations.AbstractHvdcAcEmulationFlowEquationTerm;
```

Comment:
You are creating a cyclic dependency. Network is not supposed to know anything about a particular equations system implementation